app/models/item.rb

```diff
@@ -6,9 +6,12 @@ class Item < ActiveRecord::Base
 
   has_attached_file :image, styles: {
       thumb: '100x100>',
-      square: '200x200#',
+      square: '200x200',
       medium: '300x300>'
-    }
+    },
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.